### PR TITLE
chore(tracking): Add Private Artwork Tracking (mobile)

### PR DESCRIPTION
**Pull request by lilyfromseattle**
This PR resolves [AMBER-592](https://artsyproduct.atlassian.net/browse/AMBER-592) 

### Description
Description

We want to add tracking to 5 behaviours found on [this spreadsheet](https://docs.google.com/spreadsheets/d/1zaFOn-bDKzYiK07sYoOYUwd7c4R5aAGzRA5oNmgZ5qs/edit#gid=0). If any of the methods do not already exist, then a separate PR will need to be made to Cohesion and to tag Louis so he can give feedback.

Some of the events were already being tracked (discussion [here](https://artsy.slack.com/archives/C05F8TNKGAV/p1714421942657479)) so I left those as is

This PR also includes some test fixes because those tests were not running due to an incorrect filepath

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Bugfix for displaying artist bio on private artwork screen - Lily

#### Dev changes
- Adds tracking events for user interaction of private artwork elements - Lily

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[AMBER-592]: https://artsyproduct.atlassian.net/browse/AMBER-592?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jpotts244 — COMMENTED** [review]
LGTM but would love to get more mobil-y folks reviewed!

Do you think we should update the change log here @lilyfromseattle ?

Maybe for:
- Bugfix for displaying artist bio on private artwork screen
- Dev changes: add tracking events for user interaction of private artwork elements?

**Comment by brainbicycle** [comment]
> LGTM but would love to get more mobil-y folks reviewed!
> 
> Do you think we should update the change log here @lilyfromseattle ?
> 
> Maybe for:
> 
> * Bugfix for displaying artist bio on private artwork screen
> * Dev changes: add tracking events for user interaction of private artwork elements?

yes please! could we update the title to a conventional commit format as well, looks good otherwise!

**Comment by jpotts244** [comment]
Added the above to the changelog section of the PR!

**Comment by jpotts244** [comment]
Hm I kind of want that message from peril to update with the changelog diff

**Review by jpotts244 — APPROVED** [review]

File: src/app/Scenes/Artwork/Components/PrivateArtwork/PrivateArtworkMetadata.tsx [hunk 1]
```diff
@@ -34,25 +37,64 @@ export const PrivateArtworkMetadata: React.FC<PrivateArtworkMetadataProps> = ({
     <>
       {!!data.conditionDescription?.details && (
         <>
-          <Expandable label="Condition" expanded={isFirstItemExpanded}>
+          <Expandable
+            trackingFunction={() => {
```
Comment (damassi) [inline src/app/Scenes/Artwork/Components/PrivateArtwork/PrivateArtworkMetadata.tsx:41]: We should update this to something a bit more idiomatic in terms of callback functions, like `onTrack`

Comment (jpotts244) [reply src/app/Scenes/Artwork/Components/PrivateArtwork/PrivateArtworkMetadata.tsx:41]: Snagging this as part of another refactor PR!

**Pull request merged**